`TestPortForwardProxy_run_connsClosed ` flakiness

## Failure

#### Link(s) to logs

https://github.com/gravitational/teleport.e/actions/runs/18403196624/job/52437063567

#### Relevant snippet

```
    portforward_test.go:457: 
        	Error Trace:	/__w/teleport.e/teleport.e/lib/kube/proxy/portforward_test.go:457
        	Error:      	Should be true
        	Test:       	TestPortForwardProxy_run_connsClosed
        	Messages:   	sourceConn streams not closed
--- FAIL: TestPortForwardProxy_run_connsClosed (0.00s)
```

--- a/lib/kube/proxy/portforward_test.go
+++ b/lib/kube/proxy/portforward_test.go
@@ -454,10 +454,10 @@ func TestPortForwardProxy_run_connsClosed(t *testing.T) {
 		}
 	}, 5*time.Second, 100*time.Millisecond, "streams werent properly removed from targetConn")
 
-	require.Eventually(t, func() bool {
-		return sourceConn.streamsClosed()
-	}, 5*time.Second, 100*time.Millisecond, "sourceConn streams not closed")
-	require.True(t, targetConn.streamsClosed(), "targetConn streams not closed")
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
+		assert.True(t, sourceConn.streamsClosed(), "sourceConn streams not closed")
+		assert.True(t, targetConn.streamsClosed(), "targetConn streams not closed")
+	}, 15*time.Second, 100*time.Millisecond, "streams not closed")
 }
 
 type fakeSPDYStream struct {

--- a/lib/kube/proxy/portforward_test.go
+++ b/lib/kube/proxy/portforward_test.go
@@ -454,7 +454,9 @@ func TestPortForwardProxy_run_connsClosed(t *testing.T) {
 		}
 	}, 5*time.Second, 100*time.Millisecond, "streams werent properly removed from targetConn")
 
-	require.True(t, sourceConn.streamsClosed(), "sourceConn streams not closed")
+	require.Eventually(t, func() bool {
+		return sourceConn.streamsClosed()
+	}, 5*time.Second, 100*time.Millisecond, "sourceConn streams not closed")
 	require.True(t, targetConn.streamsClosed(), "targetConn streams not closed")
 }
 

